smart_campus/app/templates/app/option_snippet.html

```diff
@@ -0,0 +1,6 @@
+<!-- Template Snippet loading options -->
```

Comment:
To separate this kind of utility template with others, you can add a `_` prefix to the file name (i.e. `_option_snippet.html`).